### ISSUE TITLE
[Accessibility][PDF Viewer] The "Enable accessibility for screen readers" control is visually hidden, but in the page focus order

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1745]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Use TAB to navigate to the "Enable accessibility for screen readers" control. (The control is visually hidden, and follows the "Undock" icon button.)

**Current Behavior**
The "Enable accessibility for screen readers" button is focused and announced by NVDA but is hidden visually (both the focus state and element itself). Sighted keyboard-only users will encounter a tab stop without seeing a visible control.

**Expected behavior/code**
All focusable elements (even those that pertain to screen reader users) are visible and have a visually indicated focus state. Not all screen reader users are without vision and this control should be perceivable to all.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users, keyboard-only users

**WCAG SC**
2.4.3 Focus Order

**Possible Solution**
Consider whether accessibility for screen readers can be enabled by default, rather than requiring users to enable it. 

If it is not possible to enable this setting by default, ensure this control is visible and provides a visible focus indicator.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
